src/Workspaces/CSharp/Portable/Utilities/SpeculationAnalyzer.cs

```diff
@@ -528,6 +528,17 @@ protected override bool IsInvocableExpression(SyntaxNode node)
             return false;
         }
 
+        protected override bool IsMemberAccessExpression(SyntaxNode node)
+        {
+            return node.IsKind(SyntaxKind.SimpleMemberAccessExpression);
```

Comment:
Do we need to handle [SyntaxKind.ConditionalAccessExpression](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Syntax/SyntaxKind.cs,359)s right here (or more generally in this algorithm)?